Formula/t/textract.rb

```diff
@@ -67,6 +65,11 @@ class Textract < Formula
     sha256 "7ebdbd7863a3699080a69f71ec0cd30ed9bfee70bad9acc6a8e6abe9523c78c0"
   end
 
+  resource "hatchling" do
```

Comment:
do you know where this came from? (it wouldn't be from setuptools or six)

Reply:
dropped this (don't see it in the dep tree, lmk if I missed something) and added a patch to avoid needing setuptools

Reply:
Works for me! Before I'd added `hatchling`, when I did a local build I saw an error on importing it, and so I'd added it. But you're right that it doesn't seem to be in the dep tree.